src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const operate = (callback, num) => {
+    result = callback(result, num);
+
+    return calculator;
+  };
```

Comment:
[CODE KNOWLEDGE] - When creating a method in the object, you don't need to use the `function` keyword, use the shortcut instead. Declare `operate` using the shorthand method syntax.